code/main/edge/edgeRequestWorker.brs

```diff
@@ -224,6 +246,65 @@ function _adb_EdgeRequestWorker() as object
             return identityMap
         end function,
 
+        _processResponseOnSuccess: function(edgeResponse as object) as void
+            try
+                responseString = edgeResponse.getResponseString()
+                responseJson = ParseJson(responseString)
+                if _adb_isEmptyOrInvalidMap(responseJson)
+                    _adb_logError("EdgeRequestWorker::_processResponseOnSuccess() - Failed to parse response: (" + FormatJson(responseString) + ")")
+                    return
+                end if
+
+                handles = responseJson.handle
+                if _adb_isEmptyOrInvalidArray(handles)
+                    _adb_logVerbose("EdgeRequestWorker::_processResponseOnSuccess() - Empty handles in the response.")
+                    return
+                end if
+
+                for each handle in handles
+                    if _adb_isEmptyOrInvalidMap(handle)
+                        continue for
+                    end if
+
+                    if handle.type = "locationHint:result"
+                        m._extractAndCacheLocationHint(handle)
+                    else if handle.type = "state:store"
+                        m._extractAndCacheStateStore(handle)
+                    end if
+                end for
+            catch exception
+                _adb_logError("EdgeRequestWorker::_processResponseOnSuccess() - Failed to handle edge response: (" + FormatJson(responseString) + ")")
+            end try
+        end function,
+
+        _extractAndCacheStateStore: function(handle as object) as void
+            _adb_logVerbose("EdgeRequestWorker::_extractStateStore() - Extracting state store from the response handle(" + FormatJson(handle) + ")" )
+
+            if _adb_isEmptyOrInvalidMap(handle) or _adb_isEmptyOrInvalidArray(handle.payload)
+                return
+            end if
+
+            m._konductorConfig.setStateStore(handle.payload)
+        end function,
+
+        _extractAndCacheLocationHint: function(handle as object) as void
+            _adb_logVerbose("EdgeRequestWorker::_extractLocationHint() - Extracting location hint from the response handle (" + FormatJson(handle) + ")" )
+
+            if _adb_isEmptyOrInvalidMap(handle) or _adb_isEmptyOrInvalidArray(handle.payload)
+                return
+            end if
+
+            for each payload in handle.payload
+                if _adb_isEmptyOrInvalidMap(payload)
+                    continue for
+                end if
+
+                if payload.scope = "EdgeNetwork"
```

Comment:
Can we use all lowercase letters when comparing strings?

Reply:
Updated